Specs/Data/Cesium3DTiles/Metadata/AllMetadataTypes/1.1/tileset.json

```diff
@@ -0,0 +1,336 @@
+{
+  "asset": {
+    "version": "1.0",
```

Comment:
@j9liu just to confirm, this was intentional since we'll be making the change to allow `asset.version: "1.1"` in a future PR?

Reply:
yeah, I figured it would be better to change all of these in one sweep during that PR